### PR TITLE
new plugin "location" added

### PR DESCRIPTION
**Pull request by phinum**
It gets you your current location with latitude, longitude, city, country, continent and pin code.

### CODE REVIEW

**Comment by appi147** [comment]
You need to remove .vscode/settings file

File: jarviscli/plugins/location.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import requests
+import json
+from colorama import Fore
+from plugin import alias, plugin, require
+
+
+@alias("location")
```
Comment (appi147) [inline jarviscli/plugins/location.py]: alias is not required. You are using same keyword in plugin and alias

**Comment by phinum** [comment]
> You need to remove .vscode/settings file

I am unable to remove it, plz help!

**Comment by phinum** [comment]
> @appi147
> Do you know what happened to Travis :(? Did I do something wrong when removing Python 2 from .travis.yml ([6b2eff8](https://github.com/sukeesh/Jarvis/commit/6b2eff8b0bdf4f393320b1dd37ec3ebb2881dee9), [f5b35cb](https://github.com/sukeesh/Jarvis/commit/f5b35cbdc98b369b5b357484751b23d81c37a3cf))?
> 
> @phinum
> Try `git rm .vscode/settings`.

thank you very much, I've completed the given task accordingly!

**Comment by appi147** [comment]
> @appi147
> Do you know what happened to Travis :(? Did I do something wrong when removing Python 2 from .travis.yml ([6b2eff8](https://github.com/sukeesh/Jarvis/commit/6b2eff8b0bdf4f393320b1dd37ec3ebb2881dee9), [f5b35cb](https://github.com/sukeesh/Jarvis/commit/f5b35cbdc98b369b5b357484751b23d81c37a3cf))?

Thanks for bringing it to my notice

**Comment by appi147** [comment]
This works fine.

**Comment by appi147** [comment]
It works through ip address I believe. Its good or not I don't know but it will fail for VPN. Same as umbrella plugin

**Pull request merged**